browser/brave_content_browser_client.cc

```diff
@@ -271,6 +296,16 @@ void BraveContentBrowserClient::BrowserURLHandlerCreated(
   ChromeContentBrowserClient::BrowserURLHandlerCreated(handler);
 }
 
+void BraveContentBrowserClient::RenderProcessWillLaunch(
+    content::RenderProcessHost* host) {
+  Profile* profile = Profile::FromBrowserContext(host->GetBrowserContext());
+  Profile* original_profile = profile->GetOriginalProfile();
```

Comment:
not sure why upstream does it like this, but it should be 
```content::BrowserContext* BraveRendererUpdaterFactory::GetBrowserContextToUse(
    content::BrowserContext* context) const {
  return chrome::GetBrowserContextRedirectedInIncognito(context);
}```

Reply:
ok, done